src/epidemik/EpiModel.py

```diff
@@ -93,7 +96,14 @@ def add_vaccination(self, source, target, rate, start):
         """
         self.transitions.add_edge(source, target, rate=rate, start=start)
 
-    def add_age_structure(self, matrix, population):
+    def add_age_structure(self, matrix: List, population: List) -> None:
+        """ 
+        Add a vaccination transition between two compartments
```

Comment:
Copy-paste issue. The descriptions should probably be something like "Add age structure  with a contact matrix and age-structured population"

Also, the matrix is 2D, so maybe something like List[List]? (Not sure about how to specify this)